_config.yml

```diff
@@ -140,14 +141,24 @@ defaults:
 #    / ___// ____/ __ \
 #    \__ \/ __/ / / / /
 #   ___/ / /___/ /_/ /
-#  /____/_____/\____/    Verify Website to Webmaster Tools
+#  /____/_____/\____/    Settings for the jekyll-seo-tag gem
+#                        More information » https://github.com/jekyll/jekyll-seo-tag/
+
+twitter:
+  username: phlow
+
+# See also > _data/socialmedia.yml
+social:
+  name: Phlow
+  links:
+    - http://twitter.com/phlow
+    - http://www.mixcloud.com/phlow/
+    - https://plus.google.com/u/0/+Phlow
```

Comment:
The public G+ url is: https://plus.google.com/+Phlow; `u/0/` is only for logged-in users.